scripts/versioner.yaml

```diff
@@ -15,20 +15,8 @@ packages:
       - apptainer/mpi.def
       - conda/mpi/meta.yaml
       - conda/mpi/conda_build_config.yaml
-      - conda/mpi/build.sh
+      - conda/mpi/base_build.sh
```

Comment:
Damn, we missed adding apptainer/files/mpi here

Reply:
doh!